src/sentry/static/sentry/app/components/avatar/gravatar.tsx

```diff
@@ -4,28 +4,35 @@ import qs from 'query-string';
 import styled from 'react-emotion';
 
 import ConfigStore from 'app/stores/configStore';
+import {callIfFunction} from 'app/utils/callIfFunction';
 
 import {imageStyle} from './styles';
 
-class Gravatar extends React.Component {
+type Props = {
+  remoteSize: number;
+  gravatarId?: string;
+  placeholder?: string;
+  /**
+   * Should avatar be round instead of a square
+   */
+  round?: boolean;
+};
+
+type State = {
+  MD5?: any;
```

Comment:
Is there a `@types/crypto-js` we could import and type this as?

Reply:
Or can we just type it as a Function?

Reply:
I was having trouble importing the types and *not* importing the library.